Sources/FeedbacksKit/FeedbackForm/FeedbackForm.swift

```diff
@@ -3,24 +3,29 @@ import SwiftUI
 public struct FeedbackForm: View {
 
     public struct Config {
-        let title: String?
-
-        public init(title: String?) {
-            self.title = title
+        let title: String
+		let textForegroundColor: Color
+
+        public init(
+			title: String? = nil,
+			textForegroundColor: Color? = nil
+		) {
+			self.title = title ?? "_send_a_feedback".localized
+			self.textForegroundColor = textForegroundColor ?? .blue
```

Comment:
looks like there are some indent issues

Reply:
Tabs vs spaces (the oldest one in book) :D Switched Xcode over to spaces and reformatted these files so all should be consistent.